src/rp2_common/hardware_dma/include/hardware/dma.h

```diff
@@ -761,6 +765,34 @@ inline static void dma_sniffer_set_byte_swap_enabled(bool swap) {
         hw_clear_bits(&dma_hw->sniff_ctrl, DMA_SNIFF_CTRL_BSWAP_BITS);
 }
 
+/*! \brief Enable the Sniffer output invert function
+ *  \ingroup hardware_dma
+ *
+ * Perform a bit inversion on the sniffed data output
```

Comment:
tense is a bit weird here, I would go with "If enabled, the sniff data result appears bit-reversed when read. This does not affect the way the checksum is calculated" which is roughly what the registers say

Reply:
Thanks, agreed, resolved (I hope).